### PR TITLE
Basic Repository Outline

### LINKED ISSUE #3: Basic Repository Structure
#### Description

The following are required:
- README
- CODEOWNERS
- gitignore
- standards folder
- issues template
- etc

#### Motivation

This is a brand new repository.

### PR DESCRIPTION
**Pull request by bitzoic**
#### Description

Added items needed to have a basic repository. Follows the sway-applications and sway-libs repo oultines.

Added:
* README.md
* .gitignore
* ci.yaml
* .docs folder with images
* CODEOWNERS
* .gitattributes
* Issue templates
* PR templates
* Empty Forc.toml

Closed #3

### CODE REVIEW

**Comment by bitzoic** [comment]
CI will fail until we have a standard to build

File: .github/ISSUE_TEMPLATE/new_standard.yaml [hunk 1]
```diff
@@ -0,0 +1,36 @@
+name: Requesting New Standard
+description: If you are a completely new standard then use this template!
```
Comment (Braqzen) [inline .github/ISSUE_TEMPLATE/new_standard.yaml:2]:
```suggestion
description: If you are creating a new standard then use this template!
```

File: .github/ISSUE_TEMPLATE/new_standard.yaml [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+title: "<title>"
+labels: ["New Standard"]
+body:
+  - type: textarea
+    id: motivation
+    attributes:
+      label: Motivation
+      description: Describe your standard, motivation behind it and write a specification
+      placeholder: |
+        # Abstract
```
Comment (Braqzen) [inline .github/ISSUE_TEMPLATE/new_standard.yaml:12]: Note that h1 tags will be really big

File: .github/workflows/ci.yaml
Comment (Braqzen) [inline .github/workflows/ci.yaml:1]: I would bump this to use the updated version that the apps repo has

**Review by Braqzen — COMMENTED** [review]
It's merged before I could review lol

**Pull request merged**